src/main/java/com/library/libraryexercise/controller/LibraryController.java

```diff
@@ -6,47 +6,59 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.HashMap;
-import java.util.Map;
-
-import static org.springframework.http.HttpStatus.CREATED;
+import java.util.*;
 
 @RestController
 @RequestMapping("/library")
 public class LibraryController {
 
-    private final Map<String, Book> books = new HashMap<>() {{
-        put("Harry Potter i więzień Azkabanu", new Book("Harry Potter i więzień Azkabanu", "J.K. Rowling", "Fantasy", 300));
-    }};;
+    private final List<Book> books = new ArrayList<>();
```

Comment:
You can replace it with inline initialisation` Arrays.asList` and remove unnecessary constructor.